internal/consolidator/consolidator.go

```diff
@@ -143,6 +144,9 @@ func (c *Consolidator) Consolidate(ctx context.Context) error {
 			continue
 		}
 
+		// Increment consolidated bytes counter for this node
+		metrics.ConsolidatedBytesPerNode.WithLabelValues(record.NodeID.String()).Add(float64(totalBytes))
```

Comment:
nit: probably cleaner to handle the recording keeping of this metric within the DB method. i.e. within `consolidatedTable.Add`